app/jobs/close_patients_job.rb

```diff
@@ -25,6 +25,8 @@ def perform(*_args)
             ((patient.last_date_of_exposure.beginning_of_day + ADMIN_OPTIONS['monitoring_period_days'].days) == patient.created_at.beginning_of_day)
         # If the patient was enrolled on their last day of monitoring based on their last date of exposure, specify special reason for closure
         patient[:monitoring_reason] = 'Enrolled on last day of monitoring period (system)'
+      elsif patient.updated_at <= Time.now - 30.days
```

Comment:
You've got a couple of these around this PR, try to use the `ago` where you can.

Reply:
I believe all of these instances have been addressed now